app/templating/schema_context.py

```diff
@@ -66,3 +69,49 @@ def json_and_html_safe(data):
     if isinstance(data, str):
         return escape(data.replace('\\', r'\\'))
     return data
+
+
+def _get_is_repeating_answer(answer_id):
+    return g.schema.is_repeating_answer_type(answer_id)
+
+
+def _get_answer_is_in_repeating_group(answer_id):
+    return g.schema.answer_is_in_repeating_group(answer_id)
+
+
+def _create_answers_list(answers, index_key):
+    """Creates a list of repeating answer values in the same index position
```

Comment:
Minor nitpick but I'm just looking at how Runner other places we use docstrings and I think structure wise it should be along these lines for consistency sake: 
https://github.com/ONSdigital/eq-survey-runner/blob/15ff9877b96a06681a8b263a6ea629b88f265579/app/helpers/form_helper.py#L16

Reply:
👍 have updated